### ISSUE TITLE
 Question `Config.py`

### ISSUE THREAD

**Issue by AliBaheri** [wassname/world-models-sonic-pytorch#2]
Hi, 
Just a quick question about `config.py`. I assume the curiosity module should reside in `config.py`, but I am not able to find it. Did you upload the correct file for this? Where can I find that intrinsic reward?

**Comment by wassname** [comment]
There are two config files, `world_models_sonic/config.py` with some hard coded data paths. And `world_models_sonic/helpers/deep_rl/config.py` which has the object which is configured in the notebook `04_train_PPO_v4-all_curious.ipynb` to use curiosity. 

Where is curiosity actually coded? If curiosity is set to true, it's used in [the PPO agent](https://github.com/wassname/world-models-sonic-pytorch/blob/master/world_models_sonic/helpers/deep_rl/ppo_agent.py#L119). To see where just cntrl-f for `if config.curiosity`.

If I was going to work on it more, I would test on the current state, but also test on a random sample of previous states. That way I could avoid overfitting. (Just in case your considering similar approaches).
